### PR TITLE
adding a gazebo position sniffer to get position at a high rate from …

### PR DESCRIPTION
**Pull request by frederictaillandier**
…another program

usage :
 <plugin name='gazebo_pose_sniffer_plugin' filename='libgazebo_pose_sniffer_plugin.so'>
      <pose_receiver_ip>127.0.0.1</pose_receiver_ip>
      <pose_receiver_port>7000</pose_receiver_port>
      <tracked_link>base_link</tracked_link>
      <tracked_link>cgo3_camera_link</tracked_link>
 </plugin>

### CODE REVIEW

File: src/gazebo_pose_sniffer_plugin.cpp [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    pose.systemId = 1;
+    pose.elementId = i;
+    ++i;
+  }
+}
+
+void PoseSnifferPlugin::OnUpdate(common::UpdateInfo const &updateInfo) {
+  unsigned int i = 0;
+  for (auto link : _links) {
+    auto pose = link->WorldCoGPose();
```
Comment (Jaeyoung-Lim) [inline src/gazebo_pose_sniffer_plugin.cpp]: Could you clarify the need to take the CoG pose rather than the link pose?

Comment (frederictaillandier) [reply src/gazebo_pose_sniffer_plugin.cpp]: no :D it was the first one I saw when I checked the api. I fixed it. Thanks

**Review by Jaeyoung-Lim — APPROVED** [review]
LGTM!

**Comment by frederictaillandier** [comment]
Thanks, unfortunately I cannot merge myself

**Pull request merged**